subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/BuildCacheStepTest.groovy

```diff
@@ -217,7 +217,7 @@ class BuildCacheStepTest extends StepSpec<IncrementalChangesContext> implements
 
         then:
         def ex = thrown Exception
-        ex.message == "Failed to store cache entry $cacheKeyHashCode for job ':test': store failure"
+        ex.message == "Failed to store cache entry 30a042b90a for job ':test': store failure"
```

Comment:
Is this intentional? Why not use the variable instead?

Reply:
Yes, it is. This was not changed when fixing: https://github.com/gradle/gradle/pull/19672#discussion_r790634923